src/info/filetype.rs

```diff
@@ -22,9 +22,10 @@ impl FileExtensions {
     #[allow(clippy::case_sensitive_file_extension_comparisons)]
     fn is_immediate(&self, file: &File<'_>) -> bool {
         file.name.to_lowercase().starts_with("readme") ||
+        file.name.to_lowercase().starts_with("makefile") ||
```

Comment:
Hi @tommady,
wouldn't that also match files like `makefile.txt` or `makefile123`, which are no Makefiles?

Reply:
I just followed the GNU doc default description as I mentioned in the PR.
Other requests I think is beyond this PRs scope.